buildroot-external/package/hassio/builder/hostapp.sh

```diff
@@ -26,13 +26,13 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-SUPERVISOR="homeassistant/${ARCH}-hassio-supervisor"
-DNS="homeassistant/${ARCH}-hassio-dns"
-AUDIO="homeassistant/${ARCH}-hassio-audio"
-CLI="homeassistant/${ARCH}-hassio-cli"
-MULTICAST="homeassistant/${ARCH}-hassio-multicast"
-OBSERVER="homeassistant/${ARCH}-hassio-observer"
-LANDINGPAGE="homeassistant/${MACHINE}-homeassistant:landingpage"
+SUPERVISOR=$(curl -s ${VERSION_URL} | jq -e -r '.images.supervisor' | sed -r "s/\{arch\}/${ARCH}/g")
```

Comment:
Instead of doing 13 calls now with this, add a new variable that holds the result of `curl -s ${VERSION_URL}` and use that new variable everywhere `curl -s ${VERSION_URL}` is used